src/isotopes_data.jl

```diff
@@ -0,0 +1,4847 @@
+_isotopes_data = [
+    Isotope(
+        "hydrogen",
+        "H",
+        1,
+        1,
+        99.9885,
+        1.007825031898*u,
+        1.4e-11*u,
+        1//2,
+        5.58569468,
+        0.0*b,
+        false,
+        Inf*s,
+        NaN*s
+    ),
```

Comment:
For `Element` we define a constructor taking keyword arguments (see [here](https://github.com/JuliaPhysics/PeriodicTable.jl/blob/master/src/PeriodicTable.jl#L45-L70)). I think we should do the same for `Isotope` because 1) this makes it clear what the numbers in the definitions mean and makes them easier to parse for humans and 2) to be consistent.

Reply:
Do we really need the humans to read it? This code is generated automatically from a DataFrame anyway.

But, I guess, I don't really care. Making the dictionary constructor does not require a lot of effort.